src/tests.cpp

```diff
@@ -1441,7 +1441,221 @@ TEST_CASE( "BSMHandler JSON Error Checking", "[ppm][filtering][error]" ) {
+
+    rapidjson::Document document;
+    document.Parse( jsonString );
+
+    // check that accelLong that is present is properly identified as present
+    bool accelLongPresent = false;
+    rapidjsonRedactor.searchForMemberByPath( document, "payload.data.partII.value.accelLong", accelLongPresent );
+    CHECK( accelLongPresent == true );
+
+    // check that a nonsense member is properly identified as not present
+    bool nonsenseMemberPresent = false;
```

Comment:
Same with these CHECKs, let's split them out into separate test cases to show them as individual unit tests.